cli/server_request_command.go

```diff
@@ -113,6 +121,11 @@ func configureServerRequestCommand(srv *fisk.CmdClause) {
 	healthz := req.Command("jetstream-health", "Request JetStream health status").Alias("healthz").Action(c.healthz)
 	healthz.Flag("js-enabled", "Checks that JetStream should be enabled on all servers").Short('J').BoolVar(&c.jsEnabled)
 	healthz.Flag("server-only", "Restricts the health check to the JetStream server only, do not check streams and consumers").Short('S').BoolVar(&c.jsServerOnly)
+
+	profilez := req.Command("profile", "Run a profile").Action(c.profilez)
+	profilez.Arg("profile", "Specify the name of the profile to run (i.e. allocs, heap, goroutine...)").StringVar(&c.profileName)
```

Comment:
lets list all the ones we know about today so `Name of the profile to run (allocs, heap, goroutine)` just expand the list to include what we have.  Users would have no way to figure this out.